packages/flutter_driver/test/src/real_tests/flutter_driver_test.dart

```diff
@@ -1193,9 +1193,9 @@ class FakeVmService extends Fake implements vms.VmService {
     commandLog.add('$method $args');
     await artificialExtensionDelay;
 
-    final vms.Response response = responses[args!['command']]!;
+    final vms.Response? response = responses[args!['command']];
     assert(response != null, 'Failed to create a response for ${args['command']}');
-    return response;
+    return response!;
```

Comment:
Is it valid for `response` to be null in this context? Previously, it had the `!` operator guaranteeing that it would never be null, even if it then checked it for null afterwards, which is admittedly mixed signals.

Reply:
It is not allowed to be null, but I think the assert here gives you a better error message than a failing `!` would give you on the previous line. So, I decided to keep the assert.